woocommerce/woocommerce-android
# Block empty billing address email submission

Summary
==========
Delivers a small fix to avoid submitting an empty email when editing the Billing address

How to Test
==========
1. Go to an Order and select the action to add or edit the Billing address
2. Leave the email field empty and try to hit `done`
3. Verify that the changes are submitted correctly and the email field is now empty


Update release notes:
